astr303a.py

```diff
@@ -0,0 +1,571 @@
+import numpy as np
```

Comment:
There are a ton of unused imports here.

Reply:
Would be helpful to have a header comment at the top that discusses what this script does, what are the inputs, what are the outputs, etc.